### ISSUE TITLE
Usage in tests, namely vue-test-utils, result in "dompurify is not a function"

### ISSUE THREAD

**Issue by semiaddict** [LeSuisse/vue-dompurify-html#1551]
I am trying to write tests using [vue-test-utils-next](https://github.com/vuejs/vue-test-utils-next/) for a simple component that uses `vue-dompurify-html@vue-next` with vue 3.
If I add vue-dompurify-html as a plugin via the `global` option as described here https://next.vue-test-utils.vuejs.org/migration/#no-more-createlocalvue, I get the following error:

```bash
 TypeError: dompurify is not a function
      at Object.buildDirective (dist/js/webpack:/vue-test-utils--dompurify-html/node_modules/vue-dompurify-html/dist/dompurify-html.js:14:1)
      at Object.install (dist/js/webpack:/vue-test-utils--dompurify-html/node_modules/vue-dompurify-html/dist/index.js:7:1)
      at Object.use (dist/js/webpack:/vue-test-utils--dompurify-html/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:3110:1)
      at mount (dist/js/webpack:/vue-test-utils--dompurify-html/node_modules/@vue/test-utils/dist/vue-test-utils.esm-bundler.js:7840:1)
      at shallowMount (dist/js/webpack:/vue-test-utils--dompurify-html/node_modules/@vue/test-utils/dist/vue-test-utils.esm-bundler.js:7911:1)
      at Context.<anonymous> (dist/js/webpack:/vue-test-utils--dompurify-html/tests/unit/example.spec.js:8:1)
```

Here is a simple example:
```js
import { expect } from "chai";
import { shallowMount } from "@vue/test-utils";
import VueDOMPurifyHTML from "vue-dompurify-html";
import App from "@/App.vue";

describe("App.vue", () => {
  it("renders html", () => {
    const wrapper = shallowMount(App, {
      global: {
        plugins: [VueDOMPurifyHTML],
      },
    });
    expect(wrapper.text()).to.include("This is a test");
  });
});
```

And here is a full reproduction example: https://github.com/semiaddict/vue-test-utils--dompurify-html

The App runs just fine with `npm run serve`, but the test fails with the above error.

I have managed to get a bit closer to the source of the issue, but can't figure out how to fix it:  
**When run inside the test, the require statement inside `node_modules/vue-dompurify-html/dist/dompurify-html.js` returns an object with a `default`property containing the function instead of returning the function directly.**

Any idea of how to solve this ?

Note: I originally submitted an [issue](https://github.com/vuejs/vue-test-utils-next/issues/1142) in `vue-test-utils-next`, but was told to check here instead.

Although this might be related to https://github.com/LeSuisse/vue-dompurify-html/issues/1174, that issue has no description!

**Comment by LeSuisse** [comment]
Thanks for the full reproduction example!
It also looks like #1521. I will take a closer look later today.

**Comment by LeSuisse** [comment]
Hello,

It looks like your repo does not reproduce the issue, `vue-dompurify-html` is not used in it.

Anyway do you mind trying with the freshly released vue-dompurify-html@3.0.0-beta.1 version? It includes some changes in the way the library is built.

**Comment by semiaddict** [comment]
Hello @LeSuisse,

> It looks like your repo does not reproduce the issue, `vue-dompurify-html` is not used in it.

Ah. I'm terribly sorry. It seems my push hadn't made it though.
I just pushed again.

> Anyway do you mind trying with the freshly released vue-dompurify-html@3.0.0-beta.1 version? It includes some changes in the way the library is built.

I also just pushed a new commit with the updated dependency.
That seems to work great :)
Thank you very much!
